x-pack/plugins/ml/public/application/routing/use_doc_title.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useCallback, useEffect } from 'react';
+import { ML_APP_NAME } from '../../../common/constants/app';
+import { MlRoute } from './router';
```

Comment:
nit pick, could be `import type`

Reply:
changed in https://github.com/elastic/kibana/pull/124125/commits/8e54e359ad883ec4ca771ec6b856b98bda6be424